tests/common/test_imcversion.py

```diff
@@ -39,6 +39,18 @@ def test_nightly_version2():
     assert_equal((version1 > version2), True)
 
 
+def test_nightly_version3():
+    version1 = ImcVersion("2.0(2cS6)")
+    version2 = ImcVersion("2.0(2c)")
+    assert_equal((version1 == version2), True)
```

Comment:
@waghswapnil I know as of today we do not parse the spin part of the component.. but should we? add a `spin_build (Bool)` and `spin_build_number(Int)` to the version check.. so that `2.0(2c) < 2.0(2cS6)`.
Don't have to do this in this PR. we can do it later.. but let us keep a note of it.